connectorx-python/src/pandas/mod.rs

```diff
@@ -1,52 +1,74 @@
 mod destination;
 mod pandas_columns;
 mod pystring;
-mod transport;
+mod transports;
 mod types;
 
 pub use self::destination::{PandasDestination, PandasPartitionDestination};
-pub use self::transport::PostgresPandasTransport;
+pub use self::transports::{PostgresPandasTransport, SqlitePandasTransport};
 pub use self::types::{PandasDType, PandasTypeSystem};
 use crate::errors::ConnectorAgentPythonError;
 use anyhow::anyhow;
+use connectorx::source_router::SourceType;
 use connectorx::{
-    sources::postgres::{Binary, PostgresSource, CSV},
+    sources::{
+        postgres::{Binary, PostgresSource, CSV},
+        sqlite::SqliteSource,
+    },
     Dispatcher,
 };
 use fehler::throws;
 use log::debug;
 use pyo3::{PyAny, Python};
 
 #[throws(ConnectorAgentPythonError)]
-pub fn write_pandas<'a>(py: Python<'a>, conn: &str, queries: &[&str], protocol: &str) -> &'a PyAny {
+pub fn write_pandas<'a>(
+    py: Python<'a>,
+    source_type: &SourceType,
```

Comment:
Can we merge source_type with conn? e.g. use a url parser to parse the conn string.
Also Postgres comes with two schemas: `postgres://` and `postgresql://`.